flake8p/hook.py

```diff
@@ -31,7 +31,19 @@ def parse_config(option_manager, cfg, cfg_dir):
     anything that may have been read from whatever other configuration
     file and read the `tool.flake8` section in `pyproject.toml` instead.
     """
-    file = Path.cwd()/'pyproject.toml'
+    args = vars(option_manager.parser.parse_args())
+    custom_config_file = args.get('pyproject_file')
+
+    # absolute path
+    if custom_config_file and custom_config_file.startswith('/'):
+        file = Path(custom_config_file)
```

Comment:
This check wouldn't work on Windows. Though I don't think we need it at all. Just `Path(custom_config_file)` should do the trick, for relative and absolute paths, I think.

I did `file = Path.cwd() / 'pyproject.toml'` further down in the code, but that was just to be explicit. `Path('pyproject.toml')` would do the same, for all I know.

Reply:
Yeah, I was wondering about wether it would work on windows, but did not have a windows machine to try it out on. I tried switching it and it seems like the tests pass without much ado, so feels like we can skip differentiating between absolute and relative paths, as you said.

There is the `Path().is_absolute()` functionality otherwise, but only works from python 3.4 and forward